satpy/writers/cf_writer.py

```diff
@@ -390,8 +390,11 @@ def _backend_versions_match():
 
 def _get_backend_versions():
     import netCDF4
+
+    # Make libnetcdf development version compatible with PEP440
+    libnetcdf_version = netCDF4.__netcdf4libversion__.replace("development", "dev")
```

Comment:
Do we want to try even harder and have some fail case (try/except) where we set it to `0.0.0`? Just so someone can use this regardless of what NetCDF C sets their version to?

Reply:
Good idea, done